master/static/migrations/20240415094734_add-delete-run-trigger.tx.up.sql

```diff
@@ -0,0 +1,16 @@
+CREATE OR REPLACE FUNCTION autoupdate_exp_best_trial_metrics_on_delete() RETURNS trigger AS $$
+BEGIN
+    WITH bt AS (
+        SELECT id, best_validation_id 
+        FROM trials 
+        WHERE experiment_id = OLD.experiment_id 
+        ORDER BY searcher_metric_value_signed LIMIT 1)
+    UPDATE experiments SET best_trial_id = bt.id FROM bt
+    WHERE experiments.id = OLD.experiment_id;
+    RETURN NEW;
+END;
+$$ LANGUAGE plpgsql;
```

Comment:
please add a test in `api_runs_intg_test.go` that specifically covers this trigger.

Reply:
Added.